core/src/components/checkbox-group/checkbox-group.stories.js

```diff
@@ -38,11 +42,30 @@ export const Default = () => `
         <input type="checkbox" class="fudis-checkbox__input" name="fudis-checkbox-3" value="baz" />
         <div class="fudis-checkbox__content">
           <div class="fudis-checkbox__content-wrapper">
-            <span class="fudis-checkbox__content__box"></span>
+            <span class="fudis-checkbox__content__box">
+                <!-- When checked, render the icon below -->
+                <!-- <span class="fudis-icon fudis-icon__color__gray-middle fudis-icon__lg fudis-icon__check"></span> -->
+            </span>
           </div>
           <span class="fudis-checkbox__content__label">Option 3</span>
         </div>
       </label>
+      <div id="fudis-guidance-id" class="fudis-guidance">
+        <div>
```

Comment:
Is this extra `div` necessary? Present also in _Required_ example. _Invalid_ and _Guidance_ examples have `<div id="fudis-fieldset-id_guidance">` but probably not necessary either?